### PR TITLE
feat(Footer): Added 'new' badge to 'Work For SEEK' footer link

### PR DESCRIPTION
**Pull request by lgshaw**
Helping to expose new or updated pages via the footer, a 'new' badge (similar to the one on the main nav) has been proposed for footer links.
Initially added for the updated 'Work for SEEK' pages.

BREAKING CHANGE:
none

EXAMPLE USAGE:

```js
{
    name: 'Work for SEEK',
    href: '/work-for-seek',
    analytics: 'toolbar:work+for+seek',
    secondary: true,
    newBadge: true
},
```

### CODE REVIEW

File: react/Pill/Pill.js [hunk 1]
```diff
@@ -28,10 +29,11 @@ export default class Pill extends Component<Props> {
 
   renderStaticPill() {
     const { children, text, className, ...restProps } = this.props;
+    const footerStyle = this.props.footer ? styles.footerBadge : null;
```
Comment (markdalgleish) [inline react/Pill/Pill.js]: The header/footer badge styles are non-standard compared to the rest of the system, so I don't think we should make this an option of `Pill`. If you hard-code this style inside of `Footer`, that should be fine.

File: react/Footer/data/connect.js [hunk 2]
```diff
@@ -8,7 +8,8 @@ export default [
     name: 'Work for SEEK',
     href: '/work-for-seek',
     analytics: 'toolbar:work+for+seek',
-    secondary: true
+    secondary: true,
+    badge: 'new'
```
Comment (markdalgleish) [inline react/Footer/data/connect.js]: Are we supporting multiple badge types? Would this be better off as `new: true`?

**Comment by kosanna** [comment]
Might we ever have not NEW badges? If not I don't see the point of the prop not being a boolean like `isNew` instead

File: react/Footer/Footer.js [hunk 3]
```diff
@@ -50,7 +50,7 @@ export default class Footer extends Component {
     const { locale, linkRenderer, authenticationStatus } = this.props;
 
     return (!specificLocale || specificLocale === locale) ?
-      <FooterLink secondary={secondary} linkRenderer={linkRenderer} authenticationStatus={authenticationStatus} key={i} {...restProps}>{name}</FooterLink> :
+      <div key={i}><FooterLink secondary={secondary} linkRenderer={linkRenderer} authenticationStatus={authenticationStatus} {...restProps}>{name}</FooterLink></div> :
```
Comment (kosanna) [inline react/Footer/Footer.js]: why do we need an extra wrapper here?

Comment (kosanna) [inline react/Footer/Footer.js]: Can use Fragment here

Comment (lgshaw) [reply react/Footer/Footer.js]: You are correct @kosanna , made obsolete with my latest changes. Removed!

**Comment by kosanna** [comment]
With minimum code looks like that, don't feel it's bad 
screen shot 2018-08-16 at 10 29 41 am

**Pull request merged**